docs/2.deploy/20.providers/vercel.md

```diff
@@ -119,9 +104,3 @@ export default defineNitroConfig({
   },
 });
 ```
-
-## Vercel edge functions
-
-**Preset:** `vercel_edge` (deprecated)
-
-We recommend migrating to the default Node.js runtime and enabling [Fluid compute](https://vercel.com/docs/functions/fluid-compute).
```

Comment:
This preset is already deprecated but exists in Nitro v2.

Lets keep it in discouraged state (with migration note), in v3 are fully removing it. 

Happy for any better wording you would think for section 😊

Reply:
Oof, my bad, I completely misunderstood what the section was. Bringing it back.